### DISCUSSION TITLE
migrate to manual-install. Can I set a random new password or do I need to extract passwords from somewhere?

### DISCUSSION THREAD

**Discussion by mio-19** [nextcloud/all-in-one#4036] [Questions] [answered]
I want to switch to manual-install. I assume that the aio mastercontaine manages passwords. Can I set new random passwords?

**Comment by szaimen** [comment]
Hi, may I ask why you want to switch to the manual-install?

**Reply by mio-19** [reply]
I need to set http proxy for nextcloud container so that the container can download packages from alpine official mirror

**Comment by mio-19** [comment, marked as answer]
I found passwords in the mastercontainer volume.  They worked. 


Is it required to use those passwords? What will happen if I set new passwords?

**Reply by szaimen** [reply]
> What will happen if I set new passwords?

You could potentially run into problems
